src/common/templates/_storage.tpl

```diff
@@ -6,6 +6,12 @@
         {{- $storageClass = .global.storageClass -}}
     {{- end -}}
 {{- end -}}
+{{/*
```

Comment:
This could be simplified by refactoring the logic to 
var = global
if local
    var = local

Reply:
I think the logic can fail if someone does not use global in their charts which is very rare.. but can happen